course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -215,91 +401,165 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "1\\. Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
+    "#### 1\\. Load data\n",
+    "Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# user code ..."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.3a"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "2a\\. Plot the following data in a single figure with three maps in one row:\n",
+    "----\n",
+    "\n",
+    "#### 2\\. Calculate the difference between scenarios for a given year\n",
     "\n",
-    " * the air temperature in the E1 scenario in 2099, \n",
-    " * the air temperature in the A1B scenario in 2099, and\n",
+    "Plot the following data in a single figure with three maps, side-by-side in one row :\n",
```

Comment:
This description needs updating now that the plotting is done is the next question.